e2e/rust/src/main.rs

```diff
@@ -15,6 +15,11 @@ fn main() {}
 pub async fn test() {
     assert!(ic_cdk::api::is_controller(&ic_cdk::caller()));
 
+    let canister_id = Principal::from_str(
+        &env::var("CANISTER_ID_EVM_RPC_STAGING").expect("Missing canister ID environment variable"),
```

Comment:
Note that this doesn't work and would probably still need to be resolved in `build.rs`.